docs/modules/ROOT/pages/configuration-format.adoc

```diff
@@ -0,0 +1,94 @@
+= Configuration Format
+:description: The Hazelcast CLC recognizes the following items in the configuration file.
+
+{description}
+
+The configuration is in YAML format.
```

Comment:
Make this a list:

* The configuration is in YAML format.
* The configuration contains the `cluster` and `ssl` sections.
* A typical configuration file is as follows:
+
yaml
:

Reply:
Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/393/commits/7a4819939ba26d23c474a41583d98bedcc2ccba4